### DISCUSSION TITLE
Team radio doesn't work

### DISCUSSION THREAD

**Discussion by carmine858** [br-g/openf1#64] [Q&A]
sorry, I would like to know if the team radio method for the latest races doesn't work for you too, when I click on the link it gives me the following error:

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
</Error>

Could you tell me how to fix the problem.

**Comment by NimmerrichterM** [comment]
Could you post the request, that you tried?

**Comment by carmine858** [comment]
hi, thanks for the reply! this is the request I use:
https://api.openf1.org/v1/team_radio?session_key=latest.
while this is one of the radio team URLs that returns to me:
https://api.openf1.org/v1/team_radio?session_key=latest


Il giorno dom 28 apr 2024 alle 15:21 NimmerrichterM <
***@***.***> ha scritto:

**Reply by NimmerrichterM** [reply]
I cen get the data, but when asked for all from latest, i only get  bahrain, when filtered with driver number. It will actually return correct session

**Reply by carmine858** [reply]
in my application i tried to do this but it doesen't work, i try to open recording url in browser but i visualize the same error.
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
</Error>.
How can i fix it?

**Comment by br-g** [comment]
Sorry there was a bug!
It has been fixed now

**Comment by carmine858** [comment]
Thank’s man great job!!!!!
Il giorno mer 1 mag 2024 alle 17:27 Bruno Godefroy ***@***.***>
ha scritto:
